GooglePay.PaymentDataCryptography/GoogleKeyProvider.cs

```diff
@@ -47,8 +48,13 @@ public class GoogleKeyProvider : ISignatureKeyProvider
 
         private readonly string _testData = null;
 
-        public GoogleKeyProvider(bool isTest = false) =>
```

Comment:
A couple of other partner were using this. Can you please create an overload chained constructor instead of removing the original one? Example: public GoogleKeyProvider(bool isTest = false) : this(isTest ? GoogleTestKeyUrl : GoogleProductionKeyUrl) { }